javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/MethodCallExprContext.java

```diff
@@ -150,12 +152,18 @@ private MethodUsage resolveMethodTypeParametersFromExplicitList(TypeSolver typeS
     @Override
     public SymbolReference<ResolvedMethodDeclaration> solveMethod(String name, List<ResolvedType> argumentsTypes, boolean staticOnly) {
         Collection<ResolvedReferenceTypeDeclaration> rrtds = findTypeDeclarations(wrappedNode.getScope());
+
+        if (rrtds.isEmpty()) {
```

Comment:
My preference would be to add a line comment here with a short note about why we're doing this and to include a link to the GitHub issue. Doing so will help any future readers of the code :)